lib/p2p/packets/types/SwapFailedPacket.ts

```diff
@@ -55,7 +57,10 @@ class SwapFailedPacket extends Packet<SwapFailedPacketBody> {
     msg.setHash(this.header.hash!);
     msg.setReqId(this.header.reqId!);
     msg.setRHash(this.body!.rHash);
-    msg.setErrorMessage(this.body!.errorMessage);
+    if (this.body!.errorMessage) {
```

Comment:
this case looks redundant, and to produce the same thing without it. If you wanted to have it anyway, you can leave it in.

Reply:
Isn't it needed to cover the case of `errorMessage` being undefined?

Reply:
AFAIK if it's undefined or an empty string then it won't affect the pb serialization anyway.

Reply:
I think you're right but my only concern is that the method signature for `setErrorMessage` expects a `string` parameter, not `string | undefined`, so I'd prefer to avoid passing `undefined` just to be safe.